nautobot/wireless/models.py

```diff
@@ -191,37 +137,38 @@ def __str__(self):
     "export_templates",
     "graphql",
 )
-class AccessPointGroupWirelessNetworkAssignment(BaseModel):
+class ControllerManagedDeviceGroupWirelessNetworkAssignment(BaseModel):
     """
-    An AccessPointGroupWirelessNetworkAssignment represents the assignment of a WirelessNetwork to an AccessPointGroup.
+    A ControllerManagedDeviceGroupWirelessNetworkAssignment represents the assignment of a WirelessNetwork to an ControllerManagedDeviceGroup.
     """
 
-    access_point_group = models.ForeignKey(
-        to="wireless.AccessPointGroup",
+    controller_managed_device_group = models.ForeignKey(
+        to="dcim.ControllerManagedDeviceGroup",
         on_delete=models.CASCADE,
         related_name="wireless_network_assignments",
     )
     wireless_network = models.ForeignKey(
         to="wireless.WirelessNetwork",
         on_delete=models.CASCADE,
-        related_name="access_point_group_assignments",
+        related_name="controller_managed_device_group_assignments",
     )
     vlan = models.ForeignKey(
         to="ipam.VLAN",
         on_delete=models.PROTECT,
-        related_name="access_point_group_wireless_network_assignments",
+        related_name="controller_managed_device_group_wireless_network_assignments",
         blank=True,
         null=True,
     )
     is_metadata_associable_model = False
-    documentation_static_path = "docs/user-guide/core-data-model/wireless/accesspointgroup.html"
+    # TODO Check if this is the correct path
```

Comment:
TODO? :-) Looks OK though

Reply:
Got me, but that path no longer exists. I will change this to the controllermanageddevicegroup.html